indent.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Install emacs: 
+#	  sudo apt-get install emacs
+#
+# Get the indentation emacs script as follows: 
+#  	wget https://raw.github.com/Glavin001/atom-beautify/master/src/beautifiers/fortran-beautifier/emacs-fortran-formating-script.lisp
+# 
+# Change to downcase: 
+#   sed 's/-upcase-/-downcase-/g' emacs-fortran-formating-script.lisp > emacs-fortran-formating-script-down.lisp
+#
+# To use it just call this script from the main LATTE folder. 
+# 	./tools/indent.sh
+
+PROGRESS_PATH=$HOME/qmd-progress/
+
+for file in $PROGRESS_PATH/src/*.F90
+do 
+  emacs -batch -l $PROGRESS_PATH/emacs-fortran-formating-script-down.lisp -f f90-batch-indent-region $file
```

Comment:
Emacs' `f90-mode` uses a few variables that control how the code is beautified (for example indentation offsets). You should also supply values for those variables that the project is using so that a developer gets consistently formatted code. I'll add this file in a separate PR.